kairos-contracts/demo-contract-tests/tests/integration_tests.rs

```diff
@@ -0,0 +1,64 @@
+mod test_fixture;
+#[cfg(test)]
+mod tests {
+    use crate::test_fixture::TestContext;
+    use casper_types::{account::AccountHash, U512};
+
+    #[test]
+    fn should_install_deposit_contract() {
+        let mut fixture: TestContext = TestContext::new();
+        fixture.install_demo_contract(fixture.account_1);
```

Comment:
The fixture should encapsulate who is the installer. As in other comments, I don't see the benefit of introducing a fixture that does not encapsulate any internal knowledge. `account_1` is clearly used as the installer across all tests. It should just be called `installer`. the `new` associated function should then call `install_demo_contract()` without an argument, and just installing it using the `installer` account. Either encapsulate, or make it parametrized and modular.